CONTRIBUTING.md

```diff
@@ -33,99 +30,111 @@ If you use [nvm](https://github.com/creationix/nvm) or [nave](https://github.com
+When creating (S)CSS files, follow the CSS Module naming convention by using the `.module.(s)css` extension.
 
-Tokens are defined in the `src/base` directory (with the exception of product-specific tokens, which are in other subdirectories). Tokens come in two layers: In `aliases.json`, all base tokens are defined with concrete values, such as colours, numbers and sizes. The other files then map those aliases to tokens for specific elements.
+#### Adding a new component
 
-> You should probably not touch `aliases.json`, as our colour palette or grid rarely changes.
+If you want to add a new component:
 
-### Sass mixins
+1. Use `bpk-component-boilerplate` to create a new skeleton React component
+2. It's unlikely you will need to create a class component. If you are in doubt, have a look at the React guidelines for component and see if you can use a function component instead: [React component guidelines](https://react.dev/reference/react/Component)
+3. Create stories - each component has a set of stories living under `examples/bpk-component-{name}/stories.ts`. Stories should cover most visual variants of a component. Read more about Storybook stories [here](https://storybook.js.org/docs/react/writing-stories/introduction)
+4. Create tests
+    - Visual regression tests - Each UI component's stories should also include a story that begins with the name `VisualTest` - these will then be picked up by Percy to run on CI
+    - Unit tests - Unit tests live in the same folder with the component's code and rely on `jest` and `React Testing Library`
+5. Update `README.md` following the boilerplate format
 
-All Sass mixins are defined in the [`bpk-mixins`](https://github.com/Skyscanner/backpack-foundations/tree/main/packages/bpk-mixins) package. The package also exposes the Sass variables from the [`@skyscanner/bpk-foundations-web`](https://github.com/Skyscanner/backpack-foundations/tree/main/packages/bpk-foundations-web) package.
+#### Contribute breaking changes
 
-If you add a new file of mixins, for example for a new *atom*, make sure you add the file to the imports in `_index.scss`.
+Anytime a change could break existing applications it's considered a breaking change. To make upgrading Backpack easier for consumers, breaking changes should follow a deprecation cycle.
 
-### React component
+In most cases, it is recommended to create a V2 component and provide a migration guide. If your breaking change either
+- requires consumers to make multiple modifications to their code to adopt the changes (e.g. renaming or removing multiple props of an API), OR
+- involves a significant alteration of the structure or API of a component in a way that keeping both APIs within one component may make the code unreadable (e.g. rewriting a component to reduce the number of files from 20 to 5)
+then you should contribute your changes as a separate V2 component.
```

Comment:
What guidence do we follow if its a visually breaking change or even wanting to experiment with a new design?

For the latter of that question we should link down to the experimentation but also have a chat as a squad about testing with visual changes too

Reply:
>I think the same guidance should apply as the goal is for consumers to not have to make too many changes to adopt the newer version AND to ensure the component API and code is not too complex to cause confusion to devs

In which case we should call this out. That was the main idea behind my point we don't actually say it follows this so can lead to people doing things differently as it doen't explicitly mention